dockerfiles/entrypoints/build.sh

```diff
@@ -2,7 +2,9 @@
 
 ../../docker/common.sh
 
-CMD="python3 -m celery -A ${CELERY_APP_NAME}.worker worker -Ofair -c 1 -Q builder,celery,default,build01,build:default,build:large -l DEBUG"
+CELERY_STATE_DIR=/var/run/celery/
+mkdir -p $CELERY_STATE_DIR
+CMD="python3 -m celery -A ${CELERY_APP_NAME}.worker worker -Ofair -c 1 -Q builder,celery,default,build01,build:default,build:large -l DEBUG --statedb=${CELERY_STATE_DIR}worker.state"
```

Comment:
We should add a comment here explaining this. Basically what you put in the PR description.